.github/Csr.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+
+# kubi private key 
+openssl ecparam -genkey -name secp521r1 -noout -out /tmp/ecdsa-key.pem
+
+# kubi public key
+openssl ec -in /tmp/ecdsa-key.pem -pubout -out /tmp/ecdsa-public.pem
+
+
+mkdir dev
+cd dev
+
+# recuperer les adresses ip des nodes
+
+kubectl get nodes -o=jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}'
+
+# ip du control plane
+
+PUBLIC_APISERVER_URL=$(kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " ")
+
+#creation du certificat pour  faire la demande de signature par kube CA
+
+# install CFSSL tools
+
+VERSION=$(curl --silent "https://api.github.com/repos/cloudflare/cfssl/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
+VNUMBER=${VERSION#"v"}
+wget https://github.com/cloudflare/cfssl/releases/download/${VERSION}/cfssljson_${VNUMBER}_linux_amd64 -O cfssljson
+chmod +x cfssljson
+sudo mv cfssljson /usr/local/bin
+
+cfssljson -version
+VERSION=$(curl --silent "https://api.github.com/repos/cloudflare/cfssl/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
+VNUMBER=${VERSION#"v"}
+wget https://github.com/cloudflare/cfssl/releases/download/${VERSION}/cfssl_${VNUMBER}_linux_amd64 -O cfssl
+chmod +x cfssl
+sudo mv cfssl /usr/local/bin
+
+# create certificat 
+cat <<EOF | cfssl genkey - | cfssljson -bare server
+{
+  "hosts": [
+    "$PUBLIC_APISERVER_URL",
```

Comment:
Wouldn't the PUBLIC_APISERVER_URL be always the same with kind?
Would it make sense to have this as static file under tests/ ?